File: DDS/programa.py

def factorial(n: int) -> int:
    resultado: int = 1
    while n > 1:
        resultado = resultado * n
        n -= 1
    return resultado


def main():
    numero: int = 5
    pi: float = 3.14
    formula = 1 / 3 + 4 * 3 + (8 / 4 * (6 - 2))
    activo: bool = True
    mensaje = "El factorial es:"
    if numero >= 0 and activo:
        print(mensaje, factorial(numero))
    elif numero == 0:
        print("cero")
    else:
        print("Numero negativo")
    for i in range(numero):
        print(i)


main()

File: DDS/Programa.java
public class Programa {

    public static int factorial(int n) {
        int resultado = 1;
        while ((n > 1)) {
            resultado = (resultado * n);
            n -= 1;
        }
        return resultado;
    }

    public static void main(String[] args) {
        int numero = 5;
        double pi = 3.14;
        var formula = ((((double)(1) / (3)) + (4 * 3)) + (((double)(8) / (4)) * (6 - 2)));
        boolean activo = true;
        var mensaje = "El factorial es:";
        if (((numero >= 0) && activo)) {
            System.out.println(mensaje + " " + factorial(numero));
        } else if ((numero == 0)) {
            System.out.println("cero");
        } else {
            System.out.println("Numero negativo");
        }
        for (int i = 0; i < numero; i++) {
            System.out.println(i);
        }
    }
}
